sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/windowExpressions.scala

```diff
@@ -675,6 +675,7 @@ case class DenseRank(children: Seq[Expression]) extends RankLike {
   override val aggBufferAttributes = rank +: orderAttrs
   override val initialValues = zero +: orderInit
   override def sql: String = "DENSE_RANK()"
```

Comment:
The default behavior of `sql` in Expression is `$prettyName(childrenStr)`.
Since Rank functions has children, it generates like this.
```
[-DENSE_RANK()-]{+dense_rank(`gen_attr_0`, `gen_attr_1`)+}
```